docs/customizing_deis/router_settings.rst

```diff
@@ -34,9 +34,10 @@ setting                                      description
 =======================================      ==================================================================================================================================================================================================================================================================================================================================
 /deis/builder/host                           host of the builder component (set by builder)
 /deis/builder/port                           port of the builder component (set by builder)
+/deis/config/\*/whitelist                    comma separated list of IPs (or CIDR) allowed to connect to the application containers (set by controller) Example: "0.0.0.0:some_optional_label,10.0.0.0/8"
```

Comment:
I know it's really weird but it's intended, otherwise the parser thinks it's `/*  */ ` comment. You can try to remove it but it won't compile.